qemu/tests/chardev_tls_encryption.py

```diff
@@ -33,6 +34,9 @@ def run(test, params, env):
     """
     clean_cmd = params["clean_cmd"]
     try:
+        pkgs = params.get("depends_pkgs")
```

Comment:
Suggest `pkgs = params.get("depends_pkgs").split()`,  because `utils_package.package_install` accept single pkg string or multiple pkgs list.

Reply:
Should use `params.objects`

Reply:
In this case, it's just one package, is it "params.objects("depends_pkgs")" much better ?